Resources/Prototypes/_Mono/Entities/SpaceArtillery/Ammo/90mm.yml

```
@@ -25,16 +25,16 @@
   - type: Projectile
     damage:
       types:
-        Structural: 1000
+        Structural: 250
   - type: Sprite
     sprite: _Mono/Objects/SpaceArtillery/630_armorpiercing_shell_casing.rsi
     layers:
     - state: base-projectile
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)